lib/lamian/raven_context_extension.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# A patch for Raven::Context class
+module Lamian::RavenContextExtension
+  # Adds current lamian log to the extra part of all raven events generated inside Lamian.run block
+  # @see https://www.rubydoc.info/gems/sentry-raven/0.9.2/Raven/Context#extra-instance_method
+  def extra
+    log = Lamian.dump(format: :txt) or return super
```

Comment:
Can you please use
```ruby
return super unless (log = Lamian.dump(format: :txt))
```
?

Or multiline

```ruby
log = Lamian.dump(format: :txt)
return super unless log
```

It is not required by code style but i like it more